lib/msf/java/rmi/builder.rb

```diff
@@ -0,0 +1,96 @@
+# -*- coding: binary -*-
+
+require 'rex/java/serialization'
+
+module Msf
+  module Java
+    module Rmi
+      module Builder
+        # Builds a RMI header stream
+        #
+        # @param opts [Hash{Symbol => <String, Fixnum>}]
```

Comment:
Since this just passes all the arguments on to `Rex::Proto::Rmi::Model::OutputHeader`, you could replace all of this with a singe line like this:
```ruby
# (see Rex::Proto::Rmi::Model::OutputHeader)
```